lib/Differentiator/ReverseModeVisitor.cpp

```diff
@@ -3219,26 +3262,42 @@ Expr* getArraySizeExpr(const ArrayType* AT, ASTContext& context,
       Expr::EvalResult evalRes;
       bool isConst =
           clad_compat::Expr_EvaluateAsConstantExpr(E, evalRes, m_Context);
-      return DelayedStoreResult{*this, Ediff,
+      return DelayedStoreResult{*this,
+                                Ediff,
+                                /*Declaration*/ nullptr,
                                 /*isConstant*/ isConst,
                                 /*isInsideLoop*/ false,
+                                /*isFnScope*/ false,
```

Comment:
```suggestion
                                /*isFnScope=*/ false,
```
Can you make the above ones follow the same rule?